src/components/PhoneInput.tsx

```diff
@@ -20,17 +22,39 @@ const PhoneInput = ({ disabled, onSubmit }: { disabled?: boolean; onSubmit?: ()
 		setFullPhone(validatedPhone.phoneNumber)
 	}, [countryCode, phoneNumber])
 
+	useEffect(() => {
```

Comment:
@denis-ottofeller Let's add some comment about what it is, for this `useEffect`, for the future.